File: PythonSolutions/level_one/second_largest.py
if __name__ =='__main__':
    largest = 0
    larger = 0
    number = int(input("Enter a number or -1 to quit the program: "))
    while number != -1:
        if number > largest:
            larger = largest
            largest = number

        if number < largest:
            if number > larger:
                larger = number
        number = int(input("Enter a number or -1 to quit the program: "))

    print(f"The Second largest is {larger}")

File: JavaScript Solution/level one/secondLargest.js
var firstLargest = 0;
var secondLargest = 0;
var number = prompt("Enter a number or -1 to quit the program")
while (number != -1) {
    if (number > firstLargest) {
        secondLargest = firstLargest
        number = firstLargest
    }
    if (number < firstLargest) {
        if (number > secondLargest) {
            number = secondLargest
        }
    }
    var number = prompt("Enter a number or -1 to quit the program")
}
console.log("The Second largest is " + secondLargest)